modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java

```diff
@@ -615,6 +619,62 @@ public void testPostSiteStructuredContent() throws Exception {
 		assertValid(postStructuredContent3);
 	}
 
+	@Override
+	@Test
+	public void testPostSiteStructuredContentBatch() throws Exception {
+		Locale locale = LocaleUtil.getDefault();
+
+		StructuredContent randomStructuredContent1 = _randomStructuredContent(
+			locale);
+
+		String batchEndpoint =
+			"headless-delivery/v1.0/sites/" + testGroup.getGroupId() +
+				"/structured-contents/batch";
+
+		JSONObject batchJobJSONObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
I think as we are generating the test in this StructuredContentResourceTest, we can get rid of the HTTPTestUtil and just use the same resource that we are using in the rest of the tests, right?

something like:

```java
structuredContentResource.postSiteStructuredContentBatchHttpResponse(......
```

Reply:
@mauricio if that works is better than what we just discussed. And we would not need to move the code to a new class.